tests/api/common/test_mark_tasks.py

```diff
@@ -616,9 +616,10 @@ def test_set_running_dag_run_to_activate_state(self, dag_run_alter_function: Cal
         self._verify_task_instance_states_remain_default(dr)
         self._verify_dag_run_dates(self.dag1, date, new_state, middle_time)  # type: ignore
 
-    def test_set_success_dag_run_to_success(self):
+    @pytest.mark.parametrize("completed_state", [State.SUCCESS, State.FAILED])
+    def test_set_success_dag_run_to_success(self, completed_state):
```

Comment:
There were two identical test before:
`test_set_failed_dag_run_to_success` and `test_set_success_dag_run_to_success`